lib/components/molecules/QuoteMedia/QuoteMedia.js

```diff
@@ -0,0 +1,44 @@
+// @flow
+/**
+ *
+ * QuoteMedia
+ *
+ */
+import React from 'react';
+import styled from 'styled-components';
+
+import type { Node } from 'react';
+import styles from './QuoteMedia.style';
+import type { Props } from './types';
+
+import Heading from '../../atoms/Heading/Heading';
+import Blockquote from '../../atoms/Blockquote/Blockquote';
+
+const QuoteMedia = ({
+  className,
+  childClassName,
```

Comment:
This childClassName naming convention doesn't make it clear where its going to apply.

Reply:
fixed